Add component IDs for impala JDBC Java plugin (#9557)
apache/skywalking: docs/en/changes/changes.md

#### OAP Server

> Add 133 134 component ID for impala jdbc java plugin.

Reviewer: ```suggestion
* Add component ID(133) for impajdbcJDBCJava agent plugin and component ID(134) for impala server.
```

Please be accurate.

#### UI

> Add impala icon for impala jdbc java plugin.

Reviewer: ```suggestion
```

I don't think you have added the icon to UI.

Author: ui repo, https://github.com/apache/skywalking-booster-ui/pull/153

Reviewer: Then we need to wait for that PR merged, and you update the UI submodule in this PR.

Author: Okey.

Reviewer: UI PR is merged, please update the git submodule.

> Add impala icon for impajdbcJDBCJava agent plugin.

Reviewer: ```suggestion
* Add impala icon for impala JDBC Java agent plugin.
```

> * Add impala icon for impala JDBC Java agent plugin.

Reviewer: ```suggestion
* Fix: tab active incorrectly, when click tab space 
* Add impala icon for impala JDBC Java agent plugin.
```

Reviewer: This is a changelog included in your submodule update.
